color-changer.js

```diff
@@ -0,0 +1,42 @@
+function getRandomUnusedHue(usedHues) {
+    let randomHue;
+    while(randomHue === undefined || usedHues[randomHue]) {
+        randomHue = Math.floor(Math.random()*360);
+    }
+    return randomHue;
+}
+
+function changeBgHue(newHue) {
+    let minLightness = 30;
+    let maxLightness = 70;
+    const randomLightness = Math.floor(Math.random() * (maxLightness - minLightness) + minLightness)
+    document.body.style.backgroundColor = `hsl(${newHue}, 50%, ${randomLightness}%)`;
```

Comment:
It's pretty cool that you used hsl here. Just wanted to ask why you chose this method over rgb or hex?